security/form-authn/src/test/java/io/quarkus/ts/security/form/authn/CsrfIT.java

```diff
@@ -0,0 +1,89 @@
+package io.quarkus.ts.security.form.authn;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.ts.security.form.authn.csrf.CsrfConfiguration;
+import io.quarkus.ts.security.form.authn.csrf.CsrfResource;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class CsrfIT {
+    public static final String FORM_FIELD_NAME = "quarkus-qe-token";
```

Comment:
since these are constants, can you put them into `CsrfConfiguration` and use them here, instead of having there strings?